workshops/templates/workshops/person_form.html

```diff
@@ -1,5 +1,11 @@
 {% extends "workshops/_page.html" %}
 
+{% load selectable_tags %}
+
```

Comment:
https://github.com/swcarpentry/amy/pull/248/files#r28722582 + lack of consistency (you previously moved `{% load crispy_forms_tags %}`, but not in this template).